hail/python/hailtop/hailctl/batch/batch_cli_utils.py

```diff
@@ -29,9 +36,32 @@ def bool_string_to_bool(bool_string):
     raise ValueError("Input could not be resolved to a bool")
 
 
-def make_formatter(name):
+def separated_format(table_data: TableData, delim: str) -> str:
+    output = io.StringIO()
+    writer = csv.writer(output, delimiter=delim)
+    header = {k: k for k in table_data[0].keys()}
+    writer.writerow(header)
+    for d in table_data:
+        writer.writerow([v for v in d.values()])
```

Comment:
This can also just be `writer.writerow(d.values())` (the CI tests pointed this out if you look at the `check_hail` step that is failing